pkg/cli/local/install.go

```diff
@@ -77,12 +82,20 @@ func InstallLocalWalrusDockerContainer() error {
 		"--privileged",
 		"-e",
 		"SERVER_SETTING_LOCAL_ENVIRONMENT_MODE=docker",
+		"-e",
+		"SERVER_ENABLE_AUTHN=false",
 		"-v",
 		"/var/run/docker.sock:/var/run/docker.sock",
-		fmt.Sprintf("sealio/walrus:%s", getLocalWalrusTag()),
-		"walrus",
-		"--enable-authn=false",
-	)
+	}
+
+	for _, env := range opts.Env {
+		runArgs = append(runArgs, "-e", env)
+	}
```

Comment:
<img width="543" alt="image" src="https://github.com/seal-io/walrus/assets/3394724/a79ccb69-7978-4736-a520-451dd1fb8aa8">

should we allow a user to overwrite `SERVER_SETTING_LOCAL_ENVIRONMENT_MODE` and `SERVER_ENABLE_AUTHN` ?

Reply:
Updated.